Modix.Services/Authorization/AuthorizationClaims.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Modix.Services.Authorization
+{
+    public enum AuthorizationClaims
```

Comment:
I feel this should also be singular.

Reply:
See above comment regarding styling conventions.